UoS-CompSci-Team/Y2-S2-Software-Development-Project
# READY: limit card distribution on dice roll by amount of cards the bank has

# Changes

When resources are given out based on dice roll, if there are not enough resources for **everyone** then nobody will get any resources and a message will be entered into game log.

# BP_gameManager

## Give Dice Roll Resources Alpha

Local Variables:
![image](https://user-images.githubusercontent.com/29107364/234528717-5d4a2500-76b7-4cc9-9cec-0ce4a5b2ee43.png)

Before / after (details below):
![image](https://user-images.githubusercontent.com/29107364/234527636-a0b36dd4-5a19-4317-886d-020ebf0971f2.png)

![image](https://user-images.githubusercontent.com/29107364/234529926-4bf069f0-c0a3-4981-b704-feddf2bad756.png)

![image](https://user-images.githubusercontent.com/29107364/234528474-528e1cf0-6992-4503-ba4a-c427c28850c9.png)

![image (1)](https://user-images.githubusercontent.com/29107364/234529115-4e6f9578-30bf-45d5-88d4-7061b06384ec.png)

All new here:
![image](https://user-images.githubusercontent.com/29107364/234529348-6ff7cf5a-ce83-4609-b753-7501cd7b654b.png)

# Test Merge

- edit bank BP to have zero of all resources
- roll a result that matches where a settlement is located
- check the game log, it should say "bank did not have resources, nobody got anything" or similar
